datasets/hk/principal_officials/hk_principal_officials.yml

```
@@ -0,0 +1,26 @@
+name: hk_principal_officials
+title: Principal Officals of Hong Kong
```

Hong Kong Principal Officials
```suggestion
title: Hong Kong Principal Officials
```